pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -99,11 +99,28 @@ export class ConfigurationCard extends React.Component {
         $("#system_information_ssh_keys").on("hide.bs.modal", () => this.host_keys_hide());
 
         this.realmd.addEventListener("changed", () => this.setState({}));
+
+        $(superuser).on("changed", () => {
+            this.systime_setup();
+        });
     }
 
     systime_setup() {
         const self = this;
 
+        if (self.server_time) {
+            self.server_time.close();
+        } else {
+            // First time setup
+
+            /* NTPSynchronized needs to be polled so we just do that
+             * always.
+             */
+            window.setInterval(function() {
+                self.server_time.poll_ntp_synchronized();
```

Comment:
Likewise, this is in the `else` branch, thus `self.server_time` is null here. In most cases the proxy should get initialized faster than in 5 seconds, but let's please not count on that.

I still think it would be cleaner to either *completely* reinitialize `self.server_time` upon privilege changes (including removal of this timer), or just re-connect to D-Bus inside serverTime.js.

Reply:
> thus self.server_time is null here.

No, it isn't... Am I missing something?

> I still think it would be cleaner to either completely reinitialize self.server_time

That is what's happening, no?

> (including removal of this timer)

I don't think that is necessary, just like we don't need to re-attach other event handlers in order for them to use the new ServerTime object. They will automatically find the new one in `self.server_time`.